src/IntercomClient.php

```diff
@@ -280,13 +280,13 @@ public function nextSearchPage($path, $query, $pages)
                 "starting_after" => $pages->next->starting_after,
             ]
         ];
-        $response = $this->sendRequest('POST', "https://api.intercom.io/$path", $options);
+        $response = $this->post($path, $options);
         return $this->handleResponse($response);
     }
 
     public function nextCursorPage($path, $starting_after)
     {
-        $response = $this->sendRequest('GET', "https://api.intercom.io/$path?starting_after=$starting_after");
+        $response = $this->get($path . "?starting_after=$starting_after");
```

Comment:
I would prefer to use the same type of concatenation in both cases here

```suggestion
        $response = $this->get($path . "?starting_after=" . $starting_after);
```